src/app/styles/category.scss

```diff
@@ -18,18 +18,8 @@
 .mat-form-field {
   width: 50%;
 }
-.button {
-  border-radius: 15px;
-  border: none;
-  color: white;
-  background-color: variables.$yonder-pink;
-  height: 50px;
-  width: 50px; //1
-  margin: 6px;
-}
-
-.disabled-button {
-  background-color: gray;
+.material-icon-category {
```

Comment:
This class should be moved to styles.scss and named icon-button in order to be used in other places too